Corona-Warn-App/src/main/res/values/recovery_certificate_attribute_strings.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" tools:ignore="MissingTranslation">
```

Comment:
can we use the strings for Test covid details screen? or should we combine them in one place such as 
`covid_certificate_attribute_strings.xml`?

Reply:
I will check and combine them if possible 👍